src/elements/common/logger/Logger.js

```diff
@@ -123,6 +127,15 @@ class Logger extends React.Component<Props> {
      */
     handlePreviewMetric = (data: Object) => {
         const { onMetric } = this.props;
+
+        if (data.type === METRIC_TYPE_UAA_PARITY_METRIC) {
+            onMetric({
+                ...data,
+                type: METRIC_TYPE_UAA_PARITY_METRIC,
+            });
+            return;
```

Comment:
❓ do we want this return? Won't this prevent `METRIC_TYPE_PREVIEW` logging below?

Reply:
shouldn't be an issue since the `METRIC_TYPE_PREVIEW` would be a separate event. Ill change this to an if/else to make it more readable.